src/Miragejs/index.test.js

```diff
@@ -0,0 +1,88 @@
+import makeServer from "./index";
+jest.mock("axios");
+
+export const BASE_URL = "http://localhost:3000/api";
```

Comment:
Need to keep this in the constants folder

Reply:
will change this while adding test cases .